plugins/infrastructure/proxmox-infrastructure/skills/proxmox-infrastructure/SKILL.md

```
@@ -1,293 +1,96 @@
-**validate_template.py** - Validate template health via API
+**Nodes:** Foxtrot, Golf, Hotel
 
-```bash
-./tools/validate_template.py --template-id 9000
-```
-
-**vm_diagnostics.py** - VM health checks
+## Quick Examples
 
+**Clone template to VM:**
 ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)